system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -91,12 +91,13 @@
     }
 
     a {
-      color: $link-secondary;
+      color: var(--newtab-link-secondary-color);
+      font-weight: bold;
       padding-left: 3px;
     }
 
     button {
-      background: $grey-10;
+      background: $input-secondary;
```

Comment:
Looks like this should be `var(--newtab-button-secondary-color)` instead of `transparent`
![image](https://user-images.githubusercontent.com/438537/38012908-03a89896-3218-11e8-80e2-b5fe222e293a.png)
![image](https://user-images.githubusercontent.com/438537/38012916-0cd44b5e-3218-11e8-90c6-8cc8d147d103.png)

Reply:
@rlr looks like this was skipped? The disclaimer's button is now just `transparent` instead of a var